options.js

```diff
@@ -22,18 +23,21 @@ function saveOptions() {
 function restoreOptions() {
     var $period = periodInput();
     var $activated = activatedInput();
+    var $blurredText = meaningHideInput();
     var $vocabularyList = vocabularyListSelect();
 
     console.log("Options restoring...");
 
     var options = {
         period: localStorage.getItem("period") || 5,
         activated: localStorage.getItem("activated") == "true",
+        blurredText: localStorage.getItem("blurredText") == "true",
```

Comment:
What about re-naming "blurredText" option name to "hideMeaning"? I think this could be more proper and clear to context.